pal/audio/web/player-dom.ts

```diff
@@ -214,9 +207,12 @@ export class AudioPlayerDOM {
         return Promise.resolve();
     }
     stop (): Promise<void> {
-        this._domAudio!.pause();
-        this._state = AudioState.STOPPED;
-        return this.seek(0);
+        return new Promise((resolve) => {
```

Comment:
What about:
```ts
const audio = this._domAudio;
if (!audio) {
    return;
}
// ............
```

Reply:
updated https://github.com/cocos-creator/engine/pull/8405/commits/9462180b6fdaf713f1703b7a597d4830547c94cd  
there is no  need to see whether `_domAudio` is defined, the AudioPlayer constructor will ensure the `_domAudio` is defined